File: examples/id/generate/go/generateAShortId.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/id"
	"os"
)

// Generate a unique ID. Defaults to uuid.
func GenerateAshortId() {
	idService := id.NewIdService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := idService.Generate(&id.GenerateRequest{
		Type: "shortid",
	})
	fmt.Println(rsp, err)
}

File: examples/id/generate/node/generateAShortId.js
const { IdService } = require("micro-js-client/id");

// Generate a unique ID. Defaults to uuid.
async function generateAshortId() {
  let idService = new IdService(process.env.MICRO_API_TOKEN);
  let rsp = await idService.generate({
    type: "shortid",
  });
  console.log(rsp);
}

generateAshortId();
